src/com/t_oster/liblasercut/LaserCutter.java

```diff
@@ -80,6 +80,8 @@ public void sendJob(LaserJob job, ProgressListener pl) throws IllegalJobExceptio
      */
     public abstract void sendJob(LaserJob job, ProgressListener pl, List<String> warnings) throws IllegalJobException, Exception;
 
+	public abstract void saveJob(LaserJob job) throws IllegalJobException, Exception;
```

Comment:
By making this abstract, every LaserCutter has to implement that feature. So either we have to implement it in all drivers now, or at least make this method throw an exception when not overriden.